inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -111,6 +113,25 @@ class XmlVisitor : public ngraph::AttributeVisitor {
     }
 };
 
+void visit_exec_graph_node(pugi::xml_node& data, std::string& node_type_name,
+                           const ngraph::Node* n) {
+    using VariantString = ngraph::VariantImpl<std::string>;
```

Comment:
nitpick: Not sure that we need this using because we use `VariantString` only in one place.

Reply:
Removed in this [commit](https://github.com/openvinotoolkit/openvino/pull/2901/commits/de45b07be1b0d058e7cabf3e67f510a378f6cf17)